hamlpy/template/loaders.py

```diff
@@ -2,28 +2,18 @@
 
 import os
 
-try:
-    from django.template import TemplateDoesNotExist
-    from django.template.loaders import filesystem, app_directories
-
-    _django_available = True
-except ImportError as e:
-    class TemplateDoesNotExist(Exception):
-        pass
-
-    _django_available = False
+from django.conf import settings
```

Comment:
Always assume Django is available - seems reasonable for a project called 'django-hamlpy' :)